### DISCUSSION TITLE
AWSCLI window will not stay open!

### DISCUSSION THREAD

**Discussion by echochalet** [aws/aws-cli#7155] [Q&A] [answered]
I just installed AWSCLIV2 on my windows 10.   When I click on the link, it closes as soon as it opens?  
In other words....the AWSCLI Window will not stay open.

Copy and paste the link below:
https://billcory-pictures.s3.us-west-2.amazonaws.com/AWS+CLI.mp4

**Comment by orozcoadrian** [comment, marked as answer]
Please try step 2 of the [instructions](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html):
> 2. To confirm the installation, open the Start menu, search for "cmd" to open a command prompt window, and at the command prompt use the "aws --version" command.

**Reply by echochalet** [reply]
Step 2 results
C:\Users\Bill Cory>aws --version
aws-cli/2.7.20 Python/3.9.11 Windows/10 exe/AMD64 prompt/off

**Reply by orozcoadrian** [reply]
great. that shows that the cli tool is installed correctly and working as expected. You now have a command prompt window where you can run aws cli commands.

For comparison, here's what mine looks like (I'm running this in the Terminal app on a mac)
(also showing a sample command i found on [stackoverflow](https://stackoverflow.com/a/70445986) that should work for you as well)
```shell
adrian_orozco@adrians-MacBook-Pro ~ % aws --version                                                               
aws-cli/2.4.7 Python/3.8.8 Darwin/21.3.0 exe/x86_64 prompt/off
adrian_orozco@adrians-MacBook-Pro ~ % aws ec2 describe-images --owners self --query 'sort_by(Images, &CreationDate)[-7:].ImageId' --output table
---------------------------
|     DescribeImages      |
+-------------------------+
|  ami-07d4ac37e3c61a59c  |
|  ami-070b7bb685953ef1d  |
|  ami-00bfce1874c6e914f  |
|  ami-0c8fac77d2f96b92c  |
|  ami-09b5ee4527d630703  |
|  ami-08dcd4f364434a783  |
+-------------------------+
```

To summarize, to use the aws cli you want to open a command prompt window, then run `aws` commands (don't use the search results in the start menu)

**Reply by echochalet** [reply]
My problem is that I cannot open the command prompt for AWS.   I installed the awscliv2 command prompt in the directory Program Files>Amazon>AWSCLIV2   
Are you saying that I don't need to open that command prompt but instead just use the Windows command prompt?    Aren't these two different commands prompts for different purposes?

**Reply by orozcoadrian** [reply]
That's how the aws cli works: you open a command prompt and then run the "aws" executable. Each time you run it you pass arguments, for whatever it is that you need.
The "cli" stands for "command line interface". It's not a "windows" app (it doesn't have a UI)

**Reply by DeepikGit** [reply]
this was very helpful , just installed and face the same scenario. got to know that its just enabled the AWS CLI in windows command prompt.

**Comment by TheeDMG** [comment]
This was very helpful! Thank you both
